packages/runtime/container-runtime/src/containerRuntime.ts

```diff
@@ -736,6 +736,9 @@ export class ContainerRuntime extends TypedEventEmitter<IContainerRuntimeEvents>
         return runtime;
     }
 
+    /**
+    * @deprecated - This will be removed in a later release.
```

Comment:
```suggestion
    * @deprecated This will be removed in a later release.
```

No dash. <https://tsdoc.org/pages/tags/deprecated/>

Reply:
It helps to reference a version of runtime when it became deprecated, such that it's easy (when going through all deprecated comments) to quickly assess if given thing can be removed or not.